### ISSUE TITLE
`NameError` in generated Ads service templates

### ISSUE THREAD

**Issue by BenRKarl** [googleapis/gapic-generator-python#882]
A recent change seems to have changed the import statement for `field_mask_pb2` but not the calling code further down in the file. The issue is that the following:

```py
from google.protobuf import field_mask_pb2 as gp_field_mask  # type: ignore
```

Is now being generated as:

```py
from google.protobuf import field_mask_pb2 # type: ignore
```

Resulting in the error:

`NameError: name 'gp_field_mask' is not defined`

It seems to only be isolated to two places in a single file, for example [here](https://github.com/googleads/google-ads-python/blob/master/google/ads/googleads/v7/services/types/google_ads_service.py#L526).

And here's the [proto definition](https://github.com/googleapis/googleapis/blob/master/google/ads/googleads/v7/services/google_ads_service.proto#L410) for that class.

**Comment by BenRKarl** [comment]
Just noticed it's also an issue with [this field](https://github.com/googleapis/googleapis/blob/master/google/ads/googleads/v7/services/batch_job_service.proto#L285) as well.

In previous versions we saw:

```py
from google.rpc import status_pb2 as gr_status  # type: ignore 
```

But now:

```py
from google.rpc import status_pb2 # type: ignore 
```

**Comment by busunkim96** [comment]
Ah sorry about that. This was fixed in #883 (latest release 0.46.3)

**Comment by BenRKarl** [comment]
Awesome! I'll close this for now but will reopen if we see any other issues after the next time we regenerate. Thanks for the quick turnaround!
